boskos/aws-janitor/resources/elb.go

```diff
@@ -26,20 +26,20 @@ import (
 	"k8s.io/klog"
 )
 
-// Clean-up ELBs
+// Clean-up Classic ELBs
 
-type LoadBalancers struct{}
+type ClassicLoadBalancers struct{}
```

Comment:
What do you think about ELB / ELBv2 instead of (Classic|)LoadBalancers, to be consistent with object naming?